src/inmanta/server/bootloader.py

```diff
@@ -77,11 +86,15 @@ class InmantaBootloader:
     # Cache field for available extensions
     AVAILABLE_EXTENSIONS: Optional[dict[str, str]] = None
 
-    def __init__(self) -> None:
+    def __init__(self, configure_logging: bool = False) -> None:
```

Comment:
Why do we need this option? Is it because we would configure logging elsewhere during normal operation but we want to be able to apply the config from tests? Could you add a comment / docstring?

Reply:
Yes. It's for testing.